cmake/SetupPackages.cmake

```diff
@@ -81,6 +81,17 @@ if (RAJA_ENABLE_CUDA)
 endif()
 
 
+if (RAJA_ENABLE_TBB)
```

Comment:
Just spoke with @davidbeckingsale, leaving this for now but it will probably change with the switch.